.github/CODEOWNERS

```diff
@@ -354,6 +355,8 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/test/functional/services/cases/ @elastic/response-ops
 /x-pack/test/functional_with_es_ssl/apps/cases/ @elastic/response-ops
 /x-pack/test/api_integration/apis/cases @elastic/response-ops
+/docs/api/cases @elastic/mlr-docs
+/x-pack/plugins/cases/docs/openapi @elastic/mlr-docs
```

Comment:
Do you mind if you put the `@elastic/response-ops` (`@elastic/response-ops @elastic/mlr-docs`) here to not block PRs that may change the openapi folder? Or do you think it should not be touched by the cases team?